packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -389,6 +395,69 @@ export class BrowserTool extends BaseBrowserTool {
           }
         }
 
+        case 'sleep_milliseconds': {
+          const defaultDuration = 1000;
+          const maxDuration = 60000;
+          let duration = input.duration ?? defaultDuration;
+          
+          // Enforce maximum duration
+          if (duration > maxDuration) {
+            console.warn(`Requested sleep duration ${duration}ms exceeds maximum of ${maxDuration}ms. Using maximum.`);
+            duration = maxDuration;
+          }
+
+          // Convert to seconds for logging
+          const seconds = Math.round(duration / 1000);
+          console.log(`⏳ Waiting for ${seconds} second${seconds !== 1 ? 's' : ''}...`);
+          
+          await this.page.waitForTimeout(duration);
+          output = `Finished waiting for ${seconds} second${seconds !== 1 ? 's' : ''}`;
+          break;
+        }
```

Comment:
Implementation for sleep_miliseconds: max time is 60 seconds and min is 1 second